tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/promo_block.html

```diff
@@ -2,18 +2,23 @@
 
 <div class="grid__promo promo-block">
     <div class="promo-block__content">
-        <h3>{{ value.title }}</h3>
-        <p>{{ value.description }}</p>
-        <a href="{{ value.get_button_link }}" class="button">{{ value.button_text }}</a>
+        {% include "patterns/atoms/section-title/section-title.html" with title=value.title classes="promo-block__title" %}
+        <p class="promo-block__description">{{ value.description }}</p>
+        <a href="{{ value.get_button_link }}" class="button promo-block__button">{{ value.button_text }}</a>
         {% if value.secondary_link %}
-            <hr>
+            <hr class="promo-block__divider" />
             {% with secondary_link=value.secondary_link.0.value %}
                 <a href="{{ secondary_link.url }}" class="promo-block__link">{{ secondary_link.text }}</a>
             {% endwith %}
         {% endif %}
     </div>
-    {% image value.image fill-430x320 as mobile_image %}
-    {% image value.image fill-520x630 as desktop_image %}
-    <div class="promo-block__image promo-block__image--mobile" style="background-image: url('{{ mobile_image.url }}')"></div>
-    <div class="promo-block__image promo-block__image--desktop" style="background-image: url('{{ desktop_image.url }}')"></div>
+    {% image value.image fill-430x320 saturation-0.6 as mobile_image %}
```

Comment:
Also, these need to use `format-webp`

Reply:
Added back in 1d9642e2efb3658f9445d6a1f9ea9f202c541866, must have gotten lost during rebase